research-app/src/App.vue

```diff
@@ -312,7 +312,7 @@ import { mapGetters, mapMutations, mapState } from "vuex";
 
 import { distance, fmtDegLat, fmtDegLon, fmtHours } from "@wwtelescope/astro";
 
-import { Source, WWTResearchAppModule } from "./store";
+import { LayerInfo, Source, WWTResearchAppModule } from "./store";
```

Comment:
Here, I find the `LayerInfo` name confusing, since my default assumption would be that it would also include imageset layers as one of the enumeration options: "information about any kind of WWT layer". But this type actually has a more specific role. It's extra-confusing in this particular case since the type *does* include an ImagesetInfo option for HiPS progressive catalogs ... just not imageset *layer* info. I'd suggest renaming to something like `CatalogLayerInfo`, with a doc-comment at the type definition explaining why the Imageset option is there.